esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -2106,8 +2106,7 @@ int EsdlBindingImpl::HandleSoapRequest(CHttpRequest* request,
             Owned<IPropertyTree> tgtcfg;
             Owned<IPropertyTree> tgtctx;
 
-            ns.clear();
```

Comment:
This should probably always happen. Let the namespace generator, which already has knowledge of scapps ensure a scapps compatible namespace if and only if the service configuration identifies itself as using the scapps namespace scheme.

Reply:
My reply on this is in the pull-level comment below.

Reply:
Based on offline conversation, this is OK as-is.